pkg/metrics/metrics.go

```diff
@@ -62,12 +63,23 @@ func NewServer(log *logrus.Entry) *Metrics {
 		},
 		[]string{"namespace", "pod", "container", "image"},
 	)
+	containerImageErrors := promauto.With(reg).NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "version_checker",
+			Name:      "version_checker_image_failures_total",
+			Help:      "Total number of errors where the version-checker was unable to get the latest upstream registry version",
```

Comment:
This shouldn't need `version_checker` at the beginning of the metric name
```suggestion
		prometheus.CounterOpts{
			Namespace: "version_checker",
			Name:      "image_failures_total",
			Help:      "Total number of errors where the version-checker was unable to get the latest upstream registry version",
```